packages/core/RNSentry.podspec

```diff
@@ -19,9 +20,10 @@ end
 
 is_new_arch_enabled = ENV["RCT_NEW_ARCH_ENABLED"] == "1"
 is_using_hermes = (ENV['USE_HERMES'] == nil && is_hermes_default) || ENV['USE_HERMES'] == '1'
-new_arch_enabled_flag = (is_new_arch_enabled ? folly_compiler_flags + " -DRCT_NEW_ARCH_ENABLED" : "")
+new_arch_enabled_flag = (is_new_arch_enabled ? " -DRCT_NEW_ARCH_ENABLED" : "")
 sentry_profiling_supported_flag = (is_profiling_supported ? " -DSENTRY_PROFILING_SUPPORTED=1" : "")
-other_cflags = "$(inherited)" + new_arch_enabled_flag + sentry_profiling_supported_flag
+new_hermes_runtime_flag = (is_new_hermes_runtime ? " -DNEW_HERMES_RUNTIME" : "")
+other_cflags = "$(inherited)" + folly_compiler_flags + new_arch_enabled_flag + sentry_profiling_supported_flag + new_hermes_runtime_flag
```

Comment:
q: Should we keep the `folly_compiler_flags` conditionally in the `new_arch_enabled_flag` above?